src/test/java/de/mediathekview/mserver/crawler/zdf/tasks/ZdfFilmDetailTaskTest.java

```diff
@@ -0,0 +1,135 @@
+            "/zdf/zdf_video_details1.json",
+            "Das Duo",
+            "Echte Kerle",
+            LocalDateTime.of(2017, 2, 1, 20, 15, 0),
+            Duration.ofHours(1).plusMinutes(27).plusSeconds(35),
+            "Der Mord an Studienrat Lampert führt \"Das Duo\" an eine Schule, an der Täter und Opfer sich vermutlich begegnet sind. In deren Umfeld suchen Clara Hertz und Marion Ahrens auch das Motiv.",
+            "https://www.zdf.de/filme/das-duo/das-duo-echte-kerle-102.html",
+            "http://localhost:8589/none/zdf/16/06/160605_echte_kerle_das_duo_neo/6/160605_echte_kerle_das_duo_neo_436k_p9v12.mp4",
+            "http://localhost:8589/none/zdf/16/06/160605_echte_kerle_das_duo_neo/6/160605_echte_kerle_das_duo_neo_1456k_p13v12.mp4",
+            "http://localhost:8589/none/zdf/16/06/160605_echte_kerle_das_duo_neo/6/160605_echte_kerle_das_duo_neo_3328k_p36v12.mp4",
+            "",
+            GeoLocations.GEO_NONE,
+            false
+        },
+        {
+            "/content/documents/zdf/kinder/jonalu/tanz-auf-dem-seil-102.json",
+            "/zdf/zdf_film_details3.json",
+            "/tmd/2/ngplayer_2_3/vod/ptmd/tivi/160301_folge25_tanzaufdemseil_jon",
+            "/zdf/zdf_video_details3.json",
+            "JoNaLu",
+            "Tanz auf dem Seil - Folge 25",
+            LocalDateTime.of(2018, 3, 11, 9, 50, 0),
+            Duration.ofMinutes(24).plusSeconds(55),
+            "Naya verliert beim Seiltanz ihre Glücksblume und alles geht schief. Kann ein anderer Glücksbringer helfen? Glühwürmchen Minou hat eine \"leuchtende\" Idee.",
+            "https://www.zdf.de/kinder/jonalu/tanz-auf-dem-seil-102.html",
+            "http://localhost:8589/dach/tivi/16/03/160301_folge25_tanzaufdemseil_jon/5/160301_folge25_tanzaufdemseil_jon_436k_p9v12.mp4",
+            "http://localhost:8589/dach/tivi/16/03/160301_folge25_tanzaufdemseil_jon/5/160301_folge25_tanzaufdemseil_jon_2328k_p35v12.mp4",
+            "http://localhost:8589/dach/tivi/16/03/160301_folge25_tanzaufdemseil_jon/5/160301_folge25_tanzaufdemseil_jon_3328k_p36v12.mp4",
+            "",
+            GeoLocations.GEO_DE_AT_CH,
+            true
+        }
+    });
+  }
+
+  private final String filmUrl;
+  private final String filmJsonFile;
+  private final String videoUrl;
+  private final String videoJsonFile;
+  private final String expectedTopic;
+  private final String expectedTitle;
+  private final LocalDateTime expectedTime;
+  private final Duration expectedDuration;
+  private final String expectedDescription;
+  private final String expectedWebsite;
+  private final String expectedUrlSmall;
+  private final String expectedUrlNormal;
+  private final String expectedUrlHd;
+  private final String expectedSubtitle;
+  private final GeoLocations expectedGeo;
+  private final boolean optimizeUrls;
+
+  public ZdfFilmDetailTaskTest(final String aFilmUrl, final String aFilmJsonFile, final String aVideoUrl, final String aVideoJsonFile,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 16 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)